README.md

```diff
@@ -19,7 +19,7 @@ In case of updating http-connector please see [Breaking changes](#breaking-chang
 ## Prerequisites
 * Java 11
 * Maven 3
-* Flink 1.15+
+* Flink 1.16+
```

Comment:
Could you tell me please why we need 1.16?
The connector does not ship any Flink dependency with it. 
As long as the Flink's connector API is ok we are good.

I was thinking about drooping 1.15 in future and actually implement cool stuff from 1.16 like Lookup cache or Async Throttling mechanism but first I wanted to explore multi version support with dedicated artifacts or/and wait for Flink 1.18

Reply:
I will revert it to 15